snuba/cli/__init__.py

```diff
@@ -1,6 +1,21 @@
+import logging
+import time
 from pkgutil import walk_packages
 
 import click
+import sentry_sdk
+
+from snuba.environment import metrics as environment_metrics
+from snuba.environment import setup_logging, setup_sentry
+from snuba.utils.metrics.wrapper import MetricsWrapper
+
+setup_sentry()
+setup_logging("INFO")
```

Comment:
Since each script is supposed to honor the `--log-level` parameter. Have you verified that you can reinitialize the logger with a different level if the script then tries to initialize it with a different level ?

Reply:
Good catch, I've set `setup_logging` to force reset logging when the function is run to override the saved log level.